src/socket.h

```diff
@@ -30,6 +30,9 @@ typedef struct _pthreads_socket_t {
 	zend_bool blocking;
 } pthreads_socket_t;
 
+#define PHP_NORMAL_READ 0x0001
+#define PHP_BINARY_READ 0x0002
```

Comment:
you might want to define Socket class constants for these. These won't be available as userland constants if ext/sockets isn't available.